File: 6-kyu/take-a-ten-minutes-walk/take-a-ten-minutes-walk.js
function isValidWalk(walks) {
  if (walks.length !== 10) return false
  const directions = {
    e: 0,
    w: 0,
    n: 0,
    s: 0
  }
  walks.map((walk) => directions[walk] = directions[walk] + 1)
  return directions.n === directions.s && directions.e === directions.w 
}

File: 6-kyu/take-a-ten-minutes-walk/take-a-ten-minutes-walk.py
def is_valid_walk(walks):
    if len(walks) != 10:
        return False
    directions = {"e":0, "w":0, "n":0, "s":0}
    for walk in walks:
        directions[walk] = directions[walk] + 1
    if directions["e"] != directions["w"] or directions["n"] != directions["s"]:
        return False
    return True;
​